modules/tensor_mechanics/src/materials/ComputeIncrementalSmallStrain.C

```diff
@@ -55,7 +57,9 @@ ComputeIncrementalSmallStrain::computeProperties()
 
     A -= Fbar; // A = grad_disp - grad_disp_old
 
-    _strain_increment[_qp] = 0.5*(A + A.transpose());
+    RankTwoTensor total_strain_increment = 0.5*(A + A.transpose());
```

Comment:
Yes, this was intentional.  If you look further down, you will see how the
temporary total_strain_increment is used.

On Thu, Nov 5, 2015 at 9:16 AM, John W. Peterson <notifications@github.com>
wrote:

> In modules/tensor_mechanics/src/materials/ComputeIncrementalSmallStrain.C
> <https://github.com/idaholab/moose/pull/5930#discussion_r44031106>:
>
> > @@ -55,7 +57,9 @@ ComputeIncrementalSmallStrain::computeProperties()
> >
> >      A -= Fbar; // A = grad_disp - grad_disp_old
> >
> > -    _strain_increment[_qp] = 0.5*(A + A.transpose());
> > +    RankTwoTensor total_strain_increment = 0.5*(A + A.transpose());
>
> Here and in one other place you've replaced 1 line with 2 lines that
> appears to me to do the same thing? Was that intended?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/5930/files#r44031106>.
>



-- 
Jason D. Hales, PhD
Department Manager
Fuels Modeling & Simulation
Idaho National Laboratory
(208) 526-2293